pandas/tests/types/test_inference.py

```diff
@@ -30,11 +30,14 @@
                                  is_float,
                                  is_bool,
                                  is_scalar,
+                                 is_scipy_sparse,
                                  _ensure_int32,
                                  _ensure_categorical)
 from pandas.types.missing import isnull
 from pandas.util import testing as tm
 
+from pandas.tests.sparse.test_frame import spmatrix  # noqa: F401
```

Comment:
maybe just put ``spmatrix`` in ``pandas/tests/sparse/common.py`` (I would use ``conftest.py`` but not sure if these are picked up correctly)